bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js

```diff
@@ -1618,7 +1618,7 @@ pimcore.document.tree = Class.create({
                     fieldLabel: t("error_page") + " (" + availableLanguages[language] + ")",
                     name: "errorDocument.localized." + language,
                     fieldCls: "input_drop_target",
-                    value: localizedErrorDocumentsData[language] ? localizedErrorDocumentsData[language] : '',
+                    value: localizedErrorDocumentsData && localizedErrorDocumentsData[language] ? localizedErrorDocumentsData[language] : '',
```

Comment:
```suggestion
                    value: (localizedErrorDocumentsData && localizedErrorDocumentsData[language]) ? localizedErrorDocumentsData[language] : '',
```
Brackets are missing, otherwise `value` would be mostly boolean, right?

Reply:
Parentheses won't make much of a difference in a tenary expression, but from the looks.. I am fine with it :)

Reply:
Can you just quickly commit the suggestion? (I'm not able to do so - it's blocked by your settings 😉). Thanks!

Reply:
Ah sorry, caching issue on GitHub 😉 All fine - thanks a lot!

Reply:
Very welcome